src/core/components/courses/UserAdditionModal.vue

```diff
@@ -74,21 +78,22 @@ export default {
 
     const input = event => emit('update:model-value', event);
 
+    const update = (event, path) => {
+      emit('update:new-user-registration', set({ ...newUserRegistration.value }, path, event));
+    };
+
     const submit = () => emit('submit');
 
     const updateUser = (event) => {
       if (Object.keys(usersCompanyOptions.value).length && usersCompanyOptions.value[event].length === 1) {
         const company = usersCompanyOptions.value[event][0].value;
-        emit('update:new-user-registration', { company, user: event });
+        const isCertified = displayIsCertified.value ? newUserRegistration.value.isCertified : null;
+        emit('update:new-user-registration', { company, user: event, ...(!isNull(isCertified) && { isCertified }) });
```

Comment:
as tu besoin d'utiliser isNull ?

Reply:
pour différencier le cas où isCertified vaut false et le cas où il vaut null, is on met juste isCertified && {isCertified} ça n'enverra pas la props si isCertified vaut false

Reply:
et mettre un `isCertified !== null` ne fonctionne pas ?

Reply:
si je peux, ça revient au même